src/main/java/eu/lucaventuri/fibry/BaseActor.java

```diff
@@ -312,7 +316,7 @@ public final void processMessages() {
                             e instanceof InterruptedException || e.getCause() instanceof InterruptedException)) {
                         Exceptions.logShort(() -> autoHealing.onInterruption.accept(e));
                     } else {
-                        System.err.println(e);
+                        logger.log(Level.FINEST, e.getMessage(), e);
```

Comment:
Please, do not print the stack trace if the original code was not doing it. I do not want to pollute the log files unnecessary.